### PR TITLE
Add a simple/quick method to retrieve LoanMetadata

### PR DESCRIPTION
**Pull request by johnholmer**

### CODE REVIEW

File: GuaranteedRate.Sextant/EncompassUtils/Reporting.cs [hunk 1]
```diff
@@ -182,5 +183,42 @@ public static IList<string> LoansLastModifiedBetween(Session session, QueryCrite
...
+
+            var fields = new StringList();
+            fields.Add("Loan.LastModified");
+            fields.Add("Loan.Guid");
+            fields.Add("Loan.LoanFolder");
+            fields.Add("Loan.LoanNumber");
+
+            try
+            {
+                // Using Reports so that we can accomodate LoanFileSeqeuenceNumber when we add it to the reporting database
```
Comment (atw-gr) [inline GuaranteedRate.Sextant/EncompassUtils/Reporting.cs]: What's the reporting database? How does `LoanFileSequenceNumber` fit into it?

File: GuaranteedRate.Sextant/EncompassUtils/Reporting.cs [hunk 2]
```diff
@@ -182,5 +183,42 @@ public static IList<string> LoansLastModifiedBetween(Session session, QueryCrite
...
+                    var item = cursor.GetItem(0);
+                    meta = new LoanMetadata
+                    {
+                        Guid = (string)item["LoanGuid"],
+                        LoanNumber = (string)item["Loan.LoanNumber"],
+                        LoanFolder = (string)item["Loan.LoanFolder"],
+                        LastModified = (DateTime)item["Loan.LastModified"]
+                    };
+                }
+                cursor.Close();
```
Comment (atw-gr) [inline GuaranteedRate.Sextant/EncompassUtils/Reporting.cs]: Does [`AutoCloseable`](https://docs.oracle.com/javase/8/docs/api/java/lang/AutoCloseable.html) exist in .NET?

File: GuaranteedRate.Sextant/EncompassUtils/Reporting.cs (hunk 1 above)
Comment (johnholmer) [reply GuaranteedRate.Sextant/EncompassUtils/Reporting.cs]: The reporting database is the subset of data that we have flagged by Encompass as reportable.  These are fields that can be exported/reported on, but not necessarily in the pipeline view.  the file sequence number would be better than a timestamp compare, but isn't in the reportable fields yet

File: GuaranteedRate.Sextant/EncompassUtils/Reporting.cs (hunk 2 above)
Comment (johnholmer) [reply GuaranteedRate.Sextant/EncompassUtils/Reporting.cs]: Closest equivalent is a "using directive" but it has to implement IDisposable, which their cursor does not

**Pull request merged**